checks/signed_tags.go

```diff
@@ -61,10 +60,18 @@ func SignedTags(c checker.Checker) checker.CheckResult {
 			return checker.RetryResult(err)
 		}
 		if gt.GetVerification().GetVerified() {
-			c.Logf("signed tag found: %s, commit: %s", t.Name, sha)
+			c.Logf("verified tag found: %s, commit: %s", t.Name, sha)
 			totalSigned++
+		} else {
+			c.Logf("!! unverified tag found: %s, commit: %s, reason: %s", t.Name, sha, gt.GetVerification().GetReason())
 		}
 	}
 
+	if totalReleases == 0 {
+		c.Logf("no releases found")
```

Comment:
i think we need to change totalReleases->totalTags and also update this message to "no tags found"

Reply:
Done